docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,55 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
+Collectd schedules data collection on network entities using management agents and protocol-specific collectors.
+
+NOTE: In the {page-component-title} docs, any reference to collectd refers to the {page-component-title} collect daemon, not the open source project of the same name.
```

Comment:
Let's say "not the independent open source project of the same name". There's still a little room for confusion since we are ourselves an open-source project, but I think adding this word mitigates it a little.

Reply:
```suggestion
NOTE: In the {page-component-title} docs, any reference to collectd refers to the {page-component-title} collect daemon, not the independent open source project of the same name.
```